src/game/items/CItem.h

```diff
@@ -94,14 +94,16 @@ class CItem : public CObjBase
 #define ATTR_SECURE				0x2000000			// Is Secure
 #define ATTR_REFORGED			0x4000000			// Is Runic Reforged.
 #define ATTR_OPENED				0x8000000			// Is Door Opened.
-#define ATTR_SHARDBOUND			0x10000000
-#define ATTR_ACCOUNTBOUND  		0x20000000
-#define ATTR_CHARACTERBOUND		0x40000000
+#define ATTR_CANUSE_PARALYZED   0x10000000          // Can able to be used while paralyzed.
 
 // TODO
 #define ATTR_CANNOTREPAIR		0x400000000000		// No repair, no fortify
 #define ATTR_FACTIONITEM		0x80000000000000	// ? Faction Item (Has cliloc)
 #define ATTR_VVVITEM			0x100000000000000	// ? Vice vs Virtue Item (Has CliLoc)
+#define ATTR_SHARDBOUND			0x200000000000000
```

Comment:
Would you mind leaving the old attrs with their value? There's no reason to change them and this could cause problems to people, now or later, if they have used those values for some other stuff

Reply:
Done 👍